File: src/main/java/kr/hyfata/zero/modules/gui/mailbox/Mailbox.java
package kr.hyfata.zero.modules.gui.mailbox;

import java.sql.Timestamp;

public class Mailbox {
    private int mailId;
    private byte[] item;
    private Timestamp expiryTime = null;
    private Timestamp sentTime = null;

    public int getMailId() {
        return mailId;
    }

    public void setMailId(int mailId) {
        this.mailId = mailId;
    }

    public byte[] getItem() {
        return item;
    }

    public void setItem(byte[] item) {
        this.item = item;
    }

    public Timestamp getExpiryTime() {
        return expiryTime;
    }

    public void setExpiryTime(Timestamp expiryTime) {
        this.expiryTime = expiryTime;
    }

    public Timestamp getSentTime() {
        return sentTime;
    }

    public void setSentTime(Timestamp sentTime) {
        this.sentTime = sentTime;
    }
}

File: src/main/java/kr/hyfata/zero/modules/mailbox/dto/Mailbox.kt
package kr.hyfata.zero.modules.mailbox.dto

import java.sql.Timestamp

class Mailbox {
    var mailId: Int = 0
    var uuid: String? = null
    var item: ByteArray? = null
    var expiryTime: Timestamp? = null
    var sentTime: Timestamp? = null
}
